blocks/quote/quote.css

```diff
@@ -0,0 +1,56 @@
+.quote-wrapper {
+  padding: 0 15px;
+  margin: 40px auto ;
+}
+
+.quote {
+  position: relative;
+  max-width: 587px;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.quote .quote-text {
+  color: var(--color-fitness);
+  font-size: 2em;
+  text-transform: uppercase;
+  font-style: italic;
+  letter-spacing: 0.05em;
+  line-height: 1.15;
+  font-weight: 500;
+  padding: 10px 0;
+  position: relative;
+}
+
+.quote .quote-author {
+  color: #999;
+  font-style: italic;
+  font-family: var(--body-font-family);
+  font-size: var(--body-font-size-card-header);
+  line-height: 1.45;
+}
+
+.quote .quote-author > div::before {
+  content: "\2014";
```

Comment:
What character is this?

Reply:
em dash

Reply:
or if that's not possible just a css comment.